### ISSUE TITLE
Remove old docs

### ISSUE THREAD

**Issue by rotu** [mateogianolio/vectorious#261]
Right now, the master branch has a copy of the docs per version. This is a pain because doing a naive search in the source tree brings up duplicate results (some outdated). I suspect the reason for this is because the `docs` subdirectory is being served at `https://docs.vectorious.org/` (which, incidentally, also lacks an `index` page).

If the docs are to be included in the source tree, they should be at an unchanging path so they can be meaningfully `blame`-d and `diff`-ed.

**Comment by mateogianolio** [comment]
Agreed, the docs are a pain atm. I have made an effort to move all docs to sphinx, if you check out the `sphinx` branch you can try it out with:

```sh
cd docs
pip install -r requirements.txt
make html
# builds html files in docs/_build
```

I still haven't figured out how to deploy it to github pages without having a copy of it in the master branch.
